files/foreman_scap_client_facts.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+
+import json
+from optparse import OptionParser
```

Comment:
Is `optparse` needed? I do not see it being used anywhere...

Reply:
@xprazak2 ack updated the code.